content/en/docs/gke/deploy/management-setup.md

```diff
@@ -168,23 +199,23 @@ deployment process, so that you can customize your management cluster if necessa
 
 Your management cluster directory contains the following files and directories:
 
-* **Makefile** is a file that define rules to automate deployment process. You can refer to [GNU make documentation](https://www.gnu.org/software/make/manual/make.html#Introduction) for more introduction. The Makefile we provide is designed to be user maintainable. You are encouraged to read, edit and maintain it to suit your own deployment customization needs.
+- **Makefile** is a file that defines rules to automate deployment process. You can refer to [GNU make documentation](https://www.gnu.org/software/make/manual/make.html#Introduction) for more introduction. The Makefile we provide is designed to be user maintainable. You are encouraged to read, edit and maintain it to suit your own deployment customization needs.
 
-* **upstream** is a directory containing kustomize packages for deploying your management cluster
+- **upstream** is a directory containing kustomize packages for deploying your management cluster
```

Comment:
Add a period at the end of the sentence. Consider making the sub list a second sentence on this list item.